elemental-iso-build

```diff
@@ -27,7 +27,7 @@ build() {
 
   rm -Rf "$TEMPDIR"
   ISONAME="elemental-$(date "+%FT%TZ")"
-  docker run --privileged --rm -v "$PWD":/mnt elemental/iso:latest build-iso -o /mnt --squash-no-compression -n "$ISONAME" --bootloader-in-rootfs --overlay-iso /iso/overlay dir:rootfs
+  docker run --privileged --rm -v "$PWD":/mnt elemental/iso:latest --config-dir . build-iso -o /mnt -n "$ISONAME" --overlay-iso /iso/overlay dir:rootfs
```

Comment:
please dont drop the `--squash-no-compression` as it saves a LOT of time in building isos, especially on github :)

Reply:
This is part of the manifest.yaml now. I placed there the non-dynamic flags.